tests/test-default-branch.json

```
@@ -0,0 +1,26 @@
+{
+    "id": "org.test.DefaultBranch",
+    "runtime": "org.gnome.Platform",
```

```suggestion
    "runtime": "org.test.Platform",
```

Didn't want to bother figuring out where the test runtime was